tvb_contrib/tvb/contrib/scripts/datatypes/time_series.py

```diff
@@ -421,7 +431,10 @@ def sample_rate(self):
 
     @property
     def space_labels(self):
-         return numpy.array(self.labels_dimensions.get(self.labels_ordering[2], []))
+        try:
+            return numpy.array(self.get_space_labels())
+        except:
```

Comment:
What exception are we expecting here?

Reply:
I corrected this now. try except is not needed and there is no get_space_labels method anymore